### ISSUE TITLE
Scrolling inside A mail from Proton Mail doesn't work

### ISSUE THREAD

**Issue by Halbour** [fastaddons/ScrollAnywhere#72]
**Describe the bug**

**Steps To Reproduce**
1. Go to webpage 'mail.proton.me'
2. Scroll over 'mail's content' with right button
3. The problem is that 'it won't scroll'

**Expected behavior**

Well - I wanted it to scroll

**Device info:**
 - operating system: Windows 11
 - Browser and version: Firefox 124.0.1
 - addon version: 9.2 (latest)

**Additional info**
Add any other info about the problem here.

**Comment by Sinaaaa** [comment]
Yes proton mail is a big problem. I just wish custom scrollbars like proton has were not a thing.

**Comment by Juraj-Masiar** [comment]
Yeah, those scrollbars are super annoying.
It may be best to report it also to Proton, and tell them to remove them.
I did implemented custom scrolling already for Google Sheet and Mega.nz pages and it was not easy at all.
Also, my ScrollAnywhere project is now so old, I'm afraid to touch it. I should build a proper tests first (again, not easy task).

And now with the AI breathing down my neck, I'm worried about my future as a programmer :(. So I'm currently focusing on other features in my "payed" addons that generates my main (yet modest) income.

**Comment by Halbour** [comment]
Hey, I'm really happy to see that you're alive, lol :)
I'm actually addicted to this extension, and the only reason I uninstalled it was because it's not updated in 2 years, and many sites are bwuggy. 
Don't worry about AI... I'm sure people will find job spots, doesn't matter what new invention will pop up. Many years ago, people were sure that they won't have jobs anymore, because cars and buses will make wages obsolete, and what will they do?
I'm sure every human, and especially a programmer, will have a work to do, at least in the foreseeable future.
I won't rush you, but... I'll just say that I'm really waiting to hear from an update someday, and until then... Good luck with work 🙏🫡

**Comment by kifzar** [comment]
Also as one guy said. People believed that with the development of phone cameras and the high resolution available today, it would lead to better or more films from a cinematic perspective, but that's not necessarily the case.
